components/XOGame.js

```diff
@@ -0,0 +1,139 @@
+import styled from "styled-components";
+import {useState} from "react";
+
+const players = {
+
+  HUMAN: {
+  SYM: "X",
+  NAME: "You",
+  },
+};
+
+export default function XOGame() {
+  
+  const [board, setBoard] = useState([
+    ["", "", ""],
+    ["", "", ""],
+    ["", "", ""],
+  ]);
+
+  function playFieldLocation(arrayIndex, index) {
+    
+    
+    board[arrayIndex][index] = players?.HUMAN?.SYM;
+    setBoard((board) => [...board]);
+    
+    
+  }
+
+  function playAgainFieldLocation() {
```

Comment:
this looks more like a function that would reset the board.

how about: 
```suggestion
  function resetPlayBoard() {
```